client/src/actions/user.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable unicorn/consistent-function-scoping */
+import axios from 'axios';
+import history from '../history';
+import { SET_USER_MESSAGE, apiURL } from '../constants';
+
+const setUserMessage = message => ({
+  message,
+  type: SET_USER_MESSAGE,
+});
+
+const DELAY = 3000;
+
+const resetMessage = () => ({
+  message: '',
+  type: SET_USER_MESSAGE,
+});
+
+export const login = ({ email, password }) => async dispatch => {
+  try {
+    const {
+      data: { Token: token },
+    } = await axios.post(
+      `${apiURL}/authenticate`,
+      JSON.stringify({
+        email,
+        password,
+      }),
+    );
+    if (token) {
+      localStorage.setItem('token', token);
```

Comment:
If possible - create the fallback for that - cookie session, since some browsers may not support local storage